src/serialmonitor/serialportctrl.ts

```diff
@@ -96,8 +100,10 @@ export class SerialPortCtrl {
         });
 
         this._child.stdout.on("data", (data) => {
-            const jsonObj = JSON.parse(data.toString())
-            this._outputChannel.append(jsonObj["payload"] + "\n");
+            if (this.isActive) {
```

Comment:
Is this fixing some other bug?

Reply:
Yes, it's helping confirm that nothing sent after the serial monitor is closed by the user.